certified-connectors/Seismic Livedoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,614 @@
+                      "rows": [
+                        [
+                          "2001-07-10",
+                          "Horace Greeley High School"
+                        ],
+                        [
+                          "2005-07-25",
+                          "Adams State College"
+                        ]
+                      ]
+                    }
+                  }
+                ],
+                "outputs": [
+                  {
+                    "format": "PPTX",
+                    "name": "Test PPTX1",
+                    "fileName": "TestPPTX1",
+                    "pptxOptions": {
+                      "imageDpi": 200,
+                      "clearNotes": false
+                    },
+                    "docxOptions": null,
+                    "pdfOptions": null,
+                    "xlsxOptions": null
+                  },
+                  {
+                    "format": "PDF",
+                    "name": "Test PDF1",
+                    "fileName": "TestPDF1",
+                    "pptxOptions": null,
+                    "docxOptions": null,
+                    "pdfOptions": {
+                      "layout": "Full Page Slides",
+                      "compatibility": "Acrobat 9.0",
+                      "openPassword": "1234",
+                      "ownerPassword": "0000",
+                      "ownerOptions": null
+                    },
+                    "xlsxOptions": null
+                  }
+                ]
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Seismic.LiveDocs.LiveDocGenSuccinctResultResp"
+            },
+            "examples": {
+              "application/json": {
+                "generatedLivedocId": "da96821e-f7cf-4bd4-a225-06d17c7e7480"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/generatedLivedocs/{generatedLivedocId}": {
+      "get": {
+        "tags": [
+          "LiveDoc"
+        ],
+        "summary": "Get status of LiveDoc generation",
+        "description": "This is where you get the status for a generation process.",
+        "operationId": "GetLiveDocGenerationStatus",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "generatedLivedocId",
+            "description": "Generated Livedoc Id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Generated Livedoc Id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Seismic.LiveDocs.LiveDocGenResultResp"
+            },
+            "examples": {
+              "application/json": {
+                "id": "da96821e-f7cf-4bd4-a225-06d17c7e7480",
```

Comment:
@rkushvaha secret look alike values are not allowed. is it possible to change this value?

Reply:
added dummy value